proto/xudrpc.proto

```diff
@@ -71,6 +71,14 @@ service Xud {
       body: "*"
     };
   }
+
+  /* Remove ban from XU node manually and connect to it. */
```

Comment:
Also noticed this, the comment says we connect to a node after unbanning it but it doesn't look like we actually do that in the code. Am I missing where we connect? If not, we should either update the description or add the logic to connect. Personally, I'd probably lean to not automatically connecting after unbanning if the logic isn't already there to do so.

Reply:
@sangaman On the issue of connecting after uban, I was thinking we could make the cli command as follows: unban <node_pub_key> <reconnect>
Where <reconnect> is a boolean that lets the user decide to reconnect or not.
Dose is that on any use?

Reply:
Yeah I think that's a good idea. If the boolean is true, then we also reconnect to the node.